### DISCUSSION TITLE
Bad .tt file creation by QT client

### DISCUSSION THREAD

**Discussion by djgraco** [BearWare/TeamTalk5#1072] [General]
Hello,
I don't know if I'm reporting a bug in the right place.
Namely, the problem is that the QT Teamtalk client does not write the server information correctly.
Suppose we have such a server
name: teamtalk5server
address: tt5.myserver.com
ports: 10333
username: user
password: pass
When we generate such an entry in the Classic client it will be generated with the correct name in the server list, line
<name>teamtalk5server</name>
but when we generate it, something like this will appear in QT
<name>user@tt5.myserver.com:10333</name>

**Comment by bear101** [comment]
You mean the file name should default to "teamtalk5server.tt" instead of "user@tt5.myserver.com:10333.tt"?

When I import the .tt file then it shows up with the name initially set in "Entry Name"

**Comment by djgraco** [comment]
Yes,
this is what I mean.
When we export the tt file its name should be saved as it appears on the server list.
